### ISSUE TITLE
Converting empty markdown string to slate issue

### ISSUE THREAD

**Issue by DianaLease** [accordproject/markdown-transform#169]
**Describe the bug**
Converting an empty markdown string to slate gives us a bad slate dom with no nodes, when we should have a text node whose text property is an empty string.

This is what we currently produce:
```
{
  "object": "value",
  "document": {
    "object": "document",
    "data": {},
    "nodes": []
  }
}
```

This is what we should produce:
```
{
  "object": "value",
  "document": {
    "object": "document",
    "data": {},
    "nodes": [
      {
        "object": "block",
        "type": "paragraph",
        "data": {},
        "nodes": [
          {
            "object": "text",
            "text": "",
            "marks": []
          }
        ]
      }
    ]
  }
}
```

**Comment by irmerk** [comment]
This is closed with #170 correct @jeromesimeon?

**Comment by DianaLease** [comment]
Yup, this now is fixed!
